scripts/data/generate_words2.py

```
@@ -0,0 +1,329 @@
+    ("prevent","防ぐ","prɪˈvent",{"3p":"prevents","past":"prevented","ing":"preventing"},"防ぐ"),
+    ("prioritize","優先する","praɪˈɔːrɪtaɪz",{"3p":"prioritizes","past":"prioritized","ing":"prioritizing"},"優先する"),
+    ("produce","生産する","prəˈduːs",{"3p":"produces","past":"produced","ing":"producing"},"生産する"),
+    ("provide","提供する","prəˈvaɪd",{"3p":"provides","past":"provided","ing":"providing"},"提供する"),
+    ("qualify","資格を得る","ˈkwɑːlɪfaɪ",{"3p":"qualifies","past":"qualified","ing":"qualifying"},"資格を得る"),
+    ("question","問いかける","ˈkwestʃən",{"3p":"questions","past":"questioned","ing":"questioning"},"問いかける"),
+    ("recommend","推薦する","ˌrekəˈmend",{"3p":"recommends","past":"recommended","ing":"recommending"},"推薦する"),
+    ("require","必要とする","rɪˈkwaɪər",{"3p":"requires","past":"required","ing":"requiring"},"必要とする"),
+    ("schedule","予定する","ˈskedʒuːl",{"3p":"schedules","past":"scheduled","ing":"scheduling"},"予定する"),
+    ("secure","確保する","sɪˈkjʊər",{"3p":"secures","past":"secured","ing":"securing"},"確保する"),
+    ("standardize","標準化する","ˈstændərdaɪz",{"3p":"standardizes","past":"standardized","ing":"standardizing"},"標準化する"),
+    ("transfer","転送する","trænsˈfɜːr",{"3p":"transfers","past":"transferred","ing":"transferring"},"転送する"),
```

Ruff (0.15.9)

[warning] String contains ambiguous `ˈ` (MODIFIER LETTER VERTICAL LINE). Did you mean ``` (GRAVE ACCENT)?

(RUF001)